bundles/bundle.properties

```diff
@@ -1,81 +1,81 @@
 # Map Info
 mod.exoprosopa.name = [#93e2ee]Exoprosopa
 mod.exoprosopa.author = TnmSpiroct
-mod.exoprosopa.description =  [#93e2ee]Exoprosopa[] adds frostnova planet into the game alongside with its own unique techtree and environment.\n\nConquer this planet with over 18 campaign maps avaiable on this mod. Including challenge maps!\n\nWith over 13 items to be discovered, 124 blocks to be researched. and 17 different units to build!\n\n[accent]Thankyou for playing this mod!\nYou can contributes maps or give suggestions by join Exoprosopa Server!\nCheck it on my profile. My username is TnmSpiroct.
+mod.exoprosopa.description =  [#93e2ee]Exoprosopa[] adds the planet Frostnova into the game alongside with its own unique techtree and environment.\n\nConquer this planet with over 18 campaign maps avaiable on this mod, including challenge maps!\n\nWith over 13 items to be discovered, 124 blocks to be researched and 17 different units to build!\n\n[accent]Thank you for playing this mod!\nYou can contribute maps or give suggestions by joining the Exoprosopa Discord Server!\nCheck it out on my profile. My username is TnmSpiroct.
```

Comment:
this is correct! Frostnova must be capitalize, it was a celestial body
refers: https://en.wikipedia.org/wiki/Capitalization_in_English#When_to_capitalize